src/main/java/graphql/annotations/processor/typeFunctions/IterableFunction.java

```diff
@@ -18,39 +18,48 @@
 import graphql.schema.GraphQLList;
 import graphql.schema.GraphQLType;
 
+import java.lang.reflect.AnnotatedArrayType;
 import java.lang.reflect.AnnotatedParameterizedType;
 import java.lang.reflect.AnnotatedType;
 import java.lang.reflect.ParameterizedType;
 
 /**
- * Support for the Iterable things like Lists / Sets / Collections and so on..
+ * Support for the Iterable things like Lists / Sets / Collections / Arrays and so on..
```

Comment:
Array is not iterable, so add a new line describing that array is also supported